external/storm-cassandra/src/main/java/org/apache/storm/cassandra/client/ClusterFactory.java

```diff
@@ -67,7 +83,57 @@ protected Cluster make(Map<String, Object> stormConf) {
                 .setConsistencyLevel(cassandraConf.getConsistencyLevel());
         cluster.withQueryOptions(options);
 
+        SslProps sslProps = cassandraConf.getSslProps();
+        configureIfSsl(cluster, sslProps);
 
         return cluster.build();
     }
+
+    /**
+     * If sslProps passed, then set SSL configuration in clusterBuilder.
+     *
+     * @param clusterBuilder cluster builder
+     * @param sslProps SSL properties
+     * @return
+     */
+    private static Builder configureIfSsl(Builder clusterBuilder, SslProps sslProps) {
+        if (sslProps == null || !sslProps.isSsl()) {
+            return clusterBuilder;
+        }
+
+        SSLContext sslContext = getSslContext(sslProps.getTruststorePath(), sslProps.getTruststorePassword(), sslProps.getKeystorePath(),
+                sslProps.getKeystorePassword());
+
+        // Default cipher suites supported by C*
```

Comment:
Any purpose for this comment being retained?

Reply:
no need for it, I'll remove it